packages/plugin-ext/src/plugin/output-channel/output-channel-item.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import * as theia from '@theia/plugin';
+import {OutputChannelRegistryMain} from '../../api/plugin-api';
+
+export class OutputChannelImpl implements theia.OutputChannel {
+
+    private _proxy: OutputChannelRegistryMain;
```

Comment:
According to https://github.com/theia-ide/theia/wiki/Coding-Guidelines: `Do not use "_" as a prefix for private properties`

Reply:
good catch